### PR TITLE
Kustomer Connector

### PR DESCRIPTION
**Pull request by SteveDMurphy**
Closes n/a

### Description of Changes

* [ ] Includes a document for the `kustomer` saas connector

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created

### CODE REVIEW

**Comment by rsilvery** [comment]
@SteveDMurphy  should I review this one or wait?

File: pages/user-guides/integrations/saas-integrations/kustomer.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+| Privacy function              | Endpoint                                                                                        | Description                                          |
+| ----------------------------- | ----------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| **Access Requests**           | [Customer Email](https://developer.kustomer.com/kustomer-api-docs/reference/getcustomerbyemail) | Returns customer profiles by email.         |
+| **Access Requests**           | [Customer Phone](https://developer.kustomer.com/kustomer-api-docs/reference/getcustomerbyphone) | Returns customer profiles by phone number.       |
+| **Update or Delete Requests** | [Delete Customer](https://developer.kustomer.com/kustomer-api-docs/reference/deletecustomer)    | Used to mask or erase customer records. |
+
+<Callout type="info" emoji="">
+By default, deletion requests are fulfilled by masking personal information. To enable Fides to personal information, ensure the `masking_strict` variable in your `fides.toml` file is set to `false`.
```
Comment (Kelsey-Ethyca) [inline pages/user-guides/integrations/saas-integrations/kustomer.mdx:44]: We are deleting, not masking so this callout is not needed

Comment (rsilvery) [reply pages/user-guides/integrations/saas-integrations/kustomer.mdx]: Updated

**Comment by rsilvery** [comment]
@Kelsey-Ethyca  updated

**Review by Kelsey-Ethyca — APPROVED** [review]

**Pull request merged**